### DISCUSSION TITLE
Summer Alarm

### DISCUSSION THREAD

**Discussion by tdorn1** [beabel/radiowecker#67] [Ideen / Ideas]
Ich habe gelesen , dass ein Buzzer definiert ist .
Kann den aber im Setup nicht aktivieren .
Kann ich also nur mit einem Sender geweckt werden ?
Ist das angedacht ? Oder muss ich den Code selber abändern ?

MFG
  Th. Dorn

**Comment by beabel** [comment]
Der Pin für den Buzzer ist festgelegt passend zum AZ-Delivery Board.
In der Software ist dieser jedoch komplett ungenutzt, da er eben nur auf dem original Board existiert.
Wenn er benutzt werden möchte muss eigener Code geschrieben und hinzugefügt werden.

**Comment by kingherold** [comment]
Servus, ich hab das bei ganz am Anfang bei V2.x selber eingebaut, weil ich mit Piep Piep geweckt werden möchte, und nicht mit Radio. Dafür habe ich den Buzzer genutzt. Mittlerweile würde ich aber gerne bei aktuellen Versionen V5.x mitmachen, weil die so viele schöne Verbesserungen hat, und nicht jedes mal den Code für den Buzzer selber einbauen. 
@beabel könnte es nicht mit integriert werden, dass man wählen kann ob Radio oder Beeper? Braucht man den Buzzer dafür, der sowieso in schlechter Qualität piept, oder kann man über die Lautsprecher nicht auch einen Audio-Piep erzeugen, da habe ich leider keine Ahnung davon. 
Aber ein Piepwecker anstelle Radio wäre schon ein Klassiker, der das Projekt steigern würde, auch wenn es dann kein "Radiowecker" mehr ist, bzw. sein muss.
LG Harry

**Reply by beabel** [reply]
@kingherold 

ich bin dran :-) 
gerade in Arbeit ist:

- Speicher optimierung (wegen LVGL wird es eng)
- eigene Wecklautstärke
- Auswahl Wecker Radio / Beep

**Reply by beabel** [reply]
V5.0.6 ist fertig und steht ab sofort zur Verfügung

**Reply by kingherold** [reply]
@beabel: Bravo!!!! konnte einwandfrei kompilieren, der Beep ist klasse! vielen Dank.

**Reply by beabel** [reply]
@kingherold 

Achtung, die lv_conf wurde geändert diese wieder in den src ordner kopieren vor dem übersetzen, da du sonst die 18'er schrift mit drin hast welche nicht mehr benutzt wird.
